SPECS-SIGNED/kernel-signed-aarch64/kernel-signed-aarch64.spec

```diff
@@ -67,7 +67,8 @@ echo "initrd of kernel %{uname_r} removed" >&2
 %postun
 if [ ! -e /boot/mariner.cfg ]
 then
-     if [ `ls /boot/linux-*.cfg 1> /dev/null 2>&1` ]
+     ls /boot/linux-*.cfg 1> /dev/null 2>&1
+     if [ $? -eq 0 ]
```

Comment:
Since we're already here, could we maybe use a single line like `if [ -f /boot/linux-*.cfg ]`? If I understod the script correctly, we're just interested if there are any files left matching the given pattern, so doing a file existence check will allow a reader to understand the intention faster.